### DISCUSSION TITLE
GDB 10.2 issue when Eclipse/CDT uses on macOS 11.4

### DISCUSSION THREAD

**Discussion by srwarner** [Homebrew/discussions#1792] [Everyday usage]
Hi,
  Is there a developer/maintainer already assigned to a formula that I should contact?
  Or am I to create pull-requests (and branches) to enabler updating the distributed version?

Sam
ps,  I've been working with the developers of GDB and Eclipse/CDT on resolving issues between the two tools.  One of the issues has been resolved in GDB over the last week.  The fix enables GDB to actually couple properly into Eclipse/CDT, and enable the IDE to operate the GDB environment; where previously GDB would fault with a null-ptr.

**Comment by carlocab** [comment]
Formulae don't have specific maintainers. If you would like to propose changes to one, please open a pull request. This guide might help: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request
